src/openrct2/actions/FootpathPlaceFromTrackAction.hpp

```diff
@@ -40,6 +40,14 @@ DEFINE_GAME_ACTION(FootpathPlaceFromTrackAction, GAME_COMMAND_PLACE_PATH_FROM_TR
     {
     }
 
+    void AcceptParameters(GameActionParameterVisitor & visitor) override
```

Comment:
Do not expose this action it should never be used by plugins it is only used by TrackDesigns.